extensions/extensions-docker/aissemble-configuration-store/src/main/resources/docker/Dockerfile

```diff
@@ -6,3 +6,17 @@ COPY --chown=default target/quarkus-app/lib/ $JAVA_APP_DIR/lib/
 COPY --chown=default target/quarkus-app/*.jar $JAVA_APP_DIR/
 COPY --chown=default target/quarkus-app/app/ $JAVA_APP_DIR/app/
 COPY --chown=default target/quarkus-app/quarkus/ $JAVA_APP_DIR/quarkus/
+
+# Custom scripts to generate certs for the mutating webhook and create the webhook itself
+COPY ./src/main/resources/scripts/genCerts.sh /usr/bin/
+COPY ./src/main/resources/scripts/genWebhook.sh /usr/bin/
+
+USER root
+
+RUN microdnf install -y openssl
+
+RUN chmod +x /usr/bin/genCerts.sh \
```

Comment:
C: These probably need to be generated at startup, so keys aren't shared.

Reply:
I just realized this is only changing the perms not actually running the script.